### PR TITLE
added a git quiz question

### PR DESCRIPTION
**Pull request by tejeswar68**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@
 /.pnp
 .pnp.js
 
+
```
Comment (jdwilkin4) [inline .gitignore:8]: This empty space here is not needed.
Please remove it.

File: .husky/pre-commit [hunk 2]
```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
+#. "$(dirname -- "$0")/_/husky.sh"
 
-npx pretty-quick --staged
+#npx pretty-quick --staged
```
Comment (jdwilkin4) [inline .husky/pre-commit:4]: Please do not make changes to this file.
Please revert back to the way it was.

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by tejeswar68** [comment]
Resolved the changes!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**